babel.config.js

```diff
@@ -0,0 +1 @@
+export const presets = ["@theforeman/builder/babel"];
```

Comment:
I guess it is the same, though usually in other plugins that's the way did it:
```suggestion
module.exports = {
  presets: ['@theforeman/builder/babel'],
};
```